### PR TITLE
Basic tests for TimeRFC3339 (un)marshaling.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: api/wireformat/entities.go [hunk 1]
```diff
@@ -117,6 +117,20 @@ func (t *TimeRFC3339) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (t *TimeRFC3339) UnmarshalYAML(unmarshal func(interface{}) error) error {
```
Comment (mattyw) [inline api/wireformat/entities.go]: godoc

**Comment by mattyw** [comment]
LGTM - but do we need TimeRFC3339 unmarshalling? It's a struct we probably should just be getting rid off

**Comment by cmars** [comment]
LGTM.

We need to keep the TimeRFC3339 type to maintain control over our wireformat representation of timestamps. For example, we don't want to output subsecond resolution, or arbitrary timezones.

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**